docs/api.md

```diff
@@ -611,6 +612,93 @@ endpoints without notice.
     issued by external issuers. These statistics only include codes issued by
     the API where an `externalIssuer` field was provided.
 
+# User report webhooks
+
+You can use your own gateway to dispatch SMS messages for user reports. When a
+user completes a self report, the verification server will send a request with
+the compiled SMS message and destination phone number to your server.
+
+- Must be a **publicly-accessible** endpoint
+- Must be **unauthenticated**
+- Must be **secured via TLS** (https)
+- Must accept a **POST** request
+- Must parse the response as JSON
+
+The request body will be identical to the [API /issue response](#apiissue). Your
```

Comment:
It can’t be identical, we need phone number (unless you added that, haven’t gotten to that file yet)

Reply:
Yea, I added it. That field is only returned if OnlyGenerateSMS is supplied though.